cmd/kubeadm/app/cmd/phases/reset/preflight.go

```diff
@@ -53,6 +55,10 @@ func runPreflight(c workflow.RunData) error {
 
 	if !r.ForceReset() && !r.DryRun() {
 		klog.Warning("[reset] WARNING: Changes made to this host by 'kubeadm init' or 'kubeadm join' will be reverted.")
+		if r.CleanupTmpDir() {
+			tempDir := path.Join(kubeadmconstants.KubernetesDir, kubeadmconstants.TempDirForKubeadm)
+			klog.Warningf("[cleanup-tmp-dir] WARNING: Cleanup the %q directory will clean up all backup data created by kubeadm, e.g. etcd data.", tempDir)
+		}
```

Comment:
```suggestion
```

what i mean in my previous comment is that we can drop the tmp dir cleanup warning entirely.
-cleanup-tmp-dir just means "add another extra dir to cleanup"

Reply:
Do we still need the check like 
```
if (!r.ForceReset() && !r.DryRun()) || r.CleanupTmpDir()) 
```

So that this flag will hit the confirmation below as well?

Reply:
i don't think so?
if -f or -dryrun are passed the prompt will not be shown. else it will be shown independent on additional flags.

Reply:
well, got you, so we don't need any change here, will update soon.